src/main/java/com/avispl/symphony/dal/infrastructure/management/polycom/lens/dto/system/QueryCost.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.infrastructure.management.polycom.lens.dto.system;
+
+/**
+ * Manage query cost in graphQL
+ * QueryCost includes queryCost, costUsed, costRemaining, secondsToReset
+ *
+ * @author Harry / Symphony Dev Team<br>
+ * Created on 4/28/2023
+ * @since 1.0.0
+ */
+public class QueryCost {
+	private Integer queryCost;
```

Comment:
Consider using primitive type instead of using the object type if we don't have to use the provided function of object type